mewarpx/mewarpx/diags_store/field_diagnostic.py

```diff
@@ -0,0 +1,55 @@
+"""Class for installing a field diagnostic with optional plotting"""
+
+from mewarpx.mwxrun import mwxrun
+from mewarpx.diags_store.diag_base import WarpXDiagnostic
+
+from pywarpx import callbacks, picmi
+from mewarpx import plotting
+
+class FieldDiagnostic(WarpXDiagnostic):
+    def __init__(self, diag_steps, diag_data_list, grid, name, write_dir,
+                 plot_on_diag_step=False, plot_data_list=None,**kwargs):
+        """
```

Comment:
docstring should be expanded: (1) Summary description of diagnostic before going into arguments. (2) indicate what valid strs for data_list are (or where a list ov valid strs) can be found (3) Describe other input arguments.